docker/Dockerfile

```diff
@@ -43,6 +44,7 @@ RUN cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
   -DENABLE_TESTING=${TESTING} -DBUILD_CSG_MANUAL=ON \
   ${COVERAGE:+-DENABLE_COVERAGE_BUILD=ON} -DBUILD_OWN_GROMACS=${BUILD_GROMACS} \
   -DENABLE_REGRESSION_TESTING=${REGRESSION_TESTING} \
+  ${CLANG_TIDY:+-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-checks=*,modernize-use-nullptr,modernize-use-override,modernize-use-equals-default,modernize-use-using,"} \
```

Comment:
I think the `*` is wrong now, maybe `-*`, but with an explicit list that seems unnecessary.